benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -25,7 +25,7 @@
     permute_dims=(False, True),
     inplace=(False, True),
     dtype=(torch.quint8, torch.qint8, torch.qint32),
-    tags=('short',)
```

Comment:
short is currently the default tag. If there is only one set of input shapes, you may want to mark it as 'short' to avoid passing tag_filter flag.

Reply:
I am adding long benchmarks in later PRs -- added short here in order not to forget :)